sql/core/src/test/resources/sql-tests/results/postgreSQL/limit.sql.out

```diff
@@ -38,6 +38,62 @@ struct<two:string,unique1:int,unique2:int,stringu1:string>
 	62	633	KCAAAA
 
 
+-- !query
+SELECT '' AS three, unique1, unique2, stringu1
+ 		FROM onek WHERE unique1 > 100
+ 		ORDER BY unique1 LIMIT 3 OFFSET 20
+-- !query schema
+struct<three:string,unique1:int,unique2:int,stringu1:string>
+-- !query output
+	121	700	REAAAA
+	122	519	SEAAAA
+	123	777	TEAAAA
+
+
+-- !query
+SELECT '' AS zero, unique1, unique2, stringu1
+ 		FROM onek WHERE unique1 < 50
+ 		ORDER BY unique1 DESC LIMIT 8 OFFSET 99
```

Comment:
could we add a few test cases with the LIMIT and OFFSET inside subqueries? Do the rows get filtered out at the table subquery boundary and then the rows from the OFFSET are not consumed by the remaining logic?

Reply:
Added into exists-orderby-limit.sql and in-limit.sql